### DISCUSSION TITLE
Support of Google Analytics

### DISCUSSION THREAD

**Discussion by frgfm** [pradyunsg/furo#401] [Potential Issue] [answered]
Hello :wave: 

Thanks for this great theme!

I couldn't find it in the documentation or the repo: is there any existing or future plan to support Google analytics? By comparison in RTD theme, we only had to add an entry in the `html_theme_options` dictionary and that would do it.

Best!

**Comment by pradyunsg** [comment, marked as answer]
There are no plans to add support for simplify injecting tracking code into the rendered documentation, at this time.

**Reply by frgfm** [reply]
Ok, got it! And can you confirm that injecting it like so https://github.com/orenhecht/googleanalytics/blob/master/sphinxcontrib/googleanalytics.py would work with this theme?

**Reply by a-r-j** [reply]
@frgfm I've been using [sphinxcontrib-gtagjs](https://pypi.org/project/sphinxcontrib-gtagjs/) and it works nicely :) You can check my [`conf.py`](https://github.com/a-r-j/graphein/blob/master/docs/source/conf.py) for an example
